### PR TITLE
Animated Navbar

### PR DESCRIPTION
**Pull request by tavisca-adsouza**

### CODE REVIEW

File: src/app/shared/components/header/header.component.ts [hunk 1]
```diff
@@ -20,23 +20,43 @@ export class HeaderComponent implements OnInit {
...
+      }
+      else{
+
+        personPin.title = this.getFirstNameLastName(this._authguardService.getUserName());
+      }
+    });
+  }
+
+  getFirstNameLastName(FullName): string{
+    var Names = FullName.split(' ');
```
Comment (lalit-tavisca) [inline src/app/shared/components/header/header.component.ts]: follow camelCase for variable names
don't use var use let or const

File: src/app/modules/zohoLogin/zoho-login/zoho-login.component.ts [hunk 2]
```diff
@@ -55,14 +57,17 @@ export class ZohoLoginComponent implements OnInit {
   }
 
   StoreLocalStroage(loginData) {
+    console.log(loginData);
+
```
Comment (lalit-tavisca) [inline src/app/modules/zohoLogin/zoho-login/zoho-login.component.ts]: remove console logs

File: src/app/app.component.html [hunk 3]
```diff
@@ -3,49 +3,38 @@
...
-                    </a>
-                    <a mat-list-item (click)="toggleActive($event)" routerLink="/plans">
-                        <mat-icon mat-list-icon>format_list_bulleted</mat-icon>
-                        <p matLine [@animateText]="_sidenavService.linkText ? 'show' : 'hide'">Training Plan</p>
-                    </a>
-                    <a [style.visibility]="_authguardService.isLoggedIn() ? 'visible' : 'hidden' " mat-list-item (click)="toggleActive($event)" routerLink="/feedback">
-                        <mat-icon id="comments" mat-list-icon>comments</mat-icon>
-                        <p matLine [@animateText]="_sidenavService.linkText ? 'show' : 'hide'">Feedback</p>
-                    </a>
+<div class="example-container" *ngIf="!isItOnLoginPage()">
```
Comment (lalit-tavisca) [inline src/app/app.component.html]: avoid method calling inside html

File: src/app/shared/components/header/header.component.ts (hunk 1 above)
Comment (tavisca-adsouza) [reply src/app/shared/components/header/header.component.ts]: Done.

File: src/app/modules/zohoLogin/zoho-login/zoho-login.component.ts (hunk 2 above)
Comment (tavisca-adsouza) [reply src/app/modules/zohoLogin/zoho-login/zoho-login.component.ts]: Removed from my code.

File: src/app/app.component.html (hunk 3 above)
Comment (tavisca-adsouza) [reply src/app/app.component.html]: Done.

**Pull request merged**